src/Transport/Amqp/AmqpTransport.php

```diff
@@ -188,17 +176,26 @@ private function createCallbackFromHandler(Closure $handler): Closure
                     try {
                         $handler($event);
                     } catch (Exception|Throwable $exception) {
-                        $this->handleFailedEvent($message, $event);
+                        $this->logError($exception->__toString());
+                        $this->handleFailedEvent($message, $event, $exception);
                     }
                 }
             } catch (DeserializationFailedException $exception) {
+                $this->logError($exception->__toString());
                 $this->handleFailedMessage($message);
             }
 
             $this->ack($message);
         };
     }
 
+    private function logError(string $message): void
+    {
+        if ($this->logger !== null) {
```

Comment:
Please write the log message to the PHP error log using `error_log()` if no logger is present

Reply:
add a error_log() call in case there is no specific logger injected